lib/mobile_robot_simulator/robot_models/differential_robot.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# Copyright (c) 2011 Autonomous Robots and Cognitive Systems Laboratory
+# Authors: Daniel Garcia Vaglio degv364@gmail.com
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# Differential Robot Class.
```

Comment:
This comment should be the Differential robot class's docstring: https://www.python.org/dev/peps/pep-0257/